packages/infra/lib/lambdas-nested-stack.ts

```
@@ -1007,6 +1031,70 @@ export class LambdasNestedStack extends NestedStack {
+    }
+
+    if (!passphraseSecret) {
+      throw new Error("LAHIE_INGESTION_PASSPHRASE is not defined in config.");
+    }
+
+    const sftpConfig = props.sftpConfig;
+
+    const lambda = createScheduledLambda({
+      layers: [ownProps.lambdaLayers.shared],
+      vpc: ownProps.vpc,
+      timeout: Duration.seconds(300),
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)